tests/functional/test_home.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.home import HomePage
+
+
+@pytest.mark.nondestructive
+def test_promos_are_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.promos.length == 16
+
+
+@pytest.mark.nondestructive
+def test_tweet_is_present(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.promos.tweet_is_present
+
+
+@pytest.mark.nondestructive
+def test_download_button_is_displayed(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    assert page.download_button_is_displayed
+
+
+@pytest.mark.nondestructive
+def test_upcoming_events_are_displayed(base_url, selenium):
+    page = HomePage(base_url, selenium).open()
+    events = page.upcoming_events
+    assert events.is_next_event_displayed
+    assert events.is_events_list_displayed
```

Comment:
Only cases where they shouldn't be displayed is if the cron fails, so I think this should probably be enough.